search/django/weedid/utils.py

```diff
@@ -118,3 +139,20 @@ def retrieve_listing_info(query_entity, awaiting_review):
         "contributor": query_entity.user.username,
         "contributor_email": query_entity.user.email if awaiting_review else "",
     }
+
+
+def parse_category_name(category):
```

Comment:
[Something like this might better belong in the weedcoco library, but it's okay here for now.]